### ISSUE TITLE
Exception: System.ArgumentOutOfRangeException:

### ISSUE THREAD

**Issue by slyfox1186** [PowerShell/PSReadLine#1545]
I am customizing my powershell prompt with the following function that I call using a separate alias.ps1 script and a defined function named `rmp`

The Function is as follows:
```
Function rmp {
  Set-Item -Path Function:prompt -Value {
    ">"
  Write-Host
  }
}
```

The error I receive upon calling the defined function shows an error output after i type a '.' (period) symbol and hit enter . This happens when the '.' is the first string in the command line + anything else that follows.

```
Last 5 Keys:
 r m p Enter
 .

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

I have another function that I use to restore the prompt using the function name 'rsp'
This function gives no issues and seems to work as intended.

This working function is shown below:

```
Function rsp {
  Set-Item -Path Function:prompt -Value {
    "$(Get-Location)" + '>'
    Write-Host
  }
  CLS
}
```
I found a similar? open issue at https://github.com/PowerShell/PSReadLine/issues/1469

I ran the code in the powershell console after recieving the error that I found on the prior link's page.

Here is the output from the code:

```
PS version: 5.1.18362.752
PSReadline version: 2.0.0-beta2
os: 10.0.18362.1 (WinBuild.160101.0800)
PS file version: 10.0.18362.1 (WinBuild.160101.0800)
HostName: ConsoleHost
BufferWidth: 160
BufferHeight: 9999
```

I wish I could help more at this time and am trying to fix this myself but I am still learning powershell. If anyone has any ideas I would greatly appreciate it. Thank you!

**Comment by daxian-dbw** [comment]
@slyfox1186 Your issue is very likely different from #1469. You are using a very old version of PSReadLine (2.0.0-beta2). The issue you ran into was highly possible fixed (see #1306).
Please upgrade to the [2.0.1 version](https://www.powershellgallery.com/packages/PSReadLine/2.0.1) of PSReadLine from PowerShellGallery and give it a go. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. If you still get the same issue after switching to 2.0.1 version of PSReadLine, please reopen this issue.

**Comment by slyfox1186** [comment]
Hey sorry for the delayed response. I followed your advice and upgraded the psreadline. Everything is now working as I feel it should be. Thank you for helping me. We can close this issue.
